docker/requirements.txt

```diff
@@ -0,0 +1,6 @@
+argparse
```

Comment:
Could you also require at least the current versions explicitly? Otherwise the use of `requirements.txt` is... Not that useful...